dpdispatcher/contexts/local_context.py

```diff
@@ -208,7 +208,10 @@ def download(
                             shutil.rmtree(lfile, ignore_errors=True)
                         elif os.path.isfile(lfile) or os.path.islink(lfile):
                             os.remove(lfile)
-                        shutil.copyfile(rfile, lfile)
+                        if not os.path.islink(rfile):
```

Comment:
Consider adding a comment explaining why the distinction between link and non-link files is necessary, as it might not be immediately clear to other developers. [medium]